src/types/outputFactory.ts

```diff
@@ -42,12 +42,15 @@ class OutputPath implements ParsedPathWithEntryPath {
   }
 
   format(): string {
-    return path.format(this)
-      // No double slashes / empty subdir name
-      .replace(/\/{2,}/g, path.sep) // unix
-      .replace(/(?<!^\\*)\\{2,}/, path.sep) // windows, preserving network paths
-      // No trailing slashes
-      .replace(/[\\/]+$/, '');
+    return (
```

Comment:
It looks like there are quite a few changes in this PR that reformat without making any functional changes. To keep things tidy, could you revert them so we only focus on the new token?

Reply:
Yeah I will take a stab at reverting the formatting changes - I would look into prettier in the future (see if you can do a clean sweep) just because its default in a lot of folks editors these days